src/math/min/index.js

```diff
@@ -0,0 +1,16 @@
+function min (array) {
+  if (!Array.isArray(array)) {
+    return undefined
+  }
+
+  if (!array.length > 0) {
```

Comment:
Bad construction. !array.length - Boolean (true|false)
In result you are checking (true|false) > 0
Change to
if (!array.length) {